### PR TITLE
Add 3rd data to plot

### PR DESCRIPTION
**Pull request by ybkamaleri**
Seems everything works and the changes doesn't affect especially plot for Fig3 🐛

### CODE REVIEW

File: ext/beta_kht/covid19.R [hunk 1]
```diff
@@ -760,6 +762,14 @@ covid19_plot_single <- function(
     d_right[, scaled_value := value / max_right * max_left]
   }
 
+  ## Third dataset
+  max_third <- max(d_third$value)
+  max_third <- max(c(max_third, 5))
+
+  if(!is.null(d_third)){
+    d_third[, scaled_value := value / max_third * max_left]
```
Comment (raubreywhite) [inline ext/beta_kht/covid19.R:770]: I think this has to be 
```
value / max_right * max_left
```
or else you are creating a third scale, which isn't correct

File: ext/beta_kht/covid19.R [hunk 2]
```diff
@@ -805,14 +815,36 @@ covid19_plot_single <- function(
...
-      color="red"
+      color="red")
+  }
+
+  if(!is.null(d_third) && !is.null(d_right)){
+    q <- q + geom_line(
+      data=d_right,
+      mapping = aes(y=scaled_value,
+                    group=1,
+                    color = "Andel NorSySS konsultasjoner"),
```
Comment (raubreywhite) [inline ext/beta_kht/covid19.R:833]: I think we should pass the color name in via arguments to the function, to make it generic. Is that possible?

File: ext/beta_kht/covid19.R [hunk 3]
```diff
@@ -805,14 +815,36 @@ covid19_plot_single <- function(
...
+      lwd = 3
+    )
+
+    q <- q + geom_line(
+      data = d_third,
+      mapping = aes(y = scaled_value,
+                    group = 1,
+                    color = "Andel positive laboratorietester"),
```
Comment (raubreywhite) [inline ext/beta_kht/covid19.R:841]: I think we should pass the color name in via arguments to the function, to make it generic. Is that possible?

File: ext/beta_kht/covid19.R [hunk 4]
```diff
@@ -869,6 +903,12 @@ covid19_plot_single <- function(
     )
   }
   q <- q + fhiplot::scale_color_fhi(labs_legend)
+
+  if(!is.null(d_third) && !is.null(d_right)){
+    q <- q + fhiplot::scale_color_fhi(palette = "posneg")
+    q <- q + guides(color = guide_legend(title = ""))
```
Comment (raubreywhite) [inline ext/beta_kht/covid19.R:909]: title = NULL

File: ext/beta_kht/covid19.R [hunk 5]
```diff
@@ -927,6 +967,19 @@ covid19_norsyss_vs_msis_daily <- function(
   setDT(d_right)
   d_right[, date:= as.Date(date)]
 
+  ## test positive
```
Comment (raubreywhite) [inline ext/beta_kht/covid19.R:970]: We can only show this graph if `location_code=="norge"`

I think in the code, where we have:

```
covid19_norsyss_vs_msis <- function(
  location_code,
  config
){
  if(get_granularity_geo(location_code) %in% c("nation", "county")){
    covid19_norsyss_vs_msis_daily(
      location_code = location_code,
      config = config
    )
  } else {
    covid19_norsyss_vs_msis_weekly(
      location_code = location_code,
      config = config
    )
  }
}
```

we need to change this to:

```
covid19_norsyss_vs_msis <- function(
  location_code,
  config
){
  if(get_granularity_geo(location_code) == "nation"){
    covid19_norsyss_vs_msis_lab_daily(
      location_code = location_code,
      config = config
    )
  } else if(get_granularity_geo(location_code) == "county") {
    covid19_norsyss_vs_msis_daily(
      location_code = location_code,
      config = config
    )
  } else {
    covid19_norsyss_vs_msis_weekly(
      location_code = location_code,
      config = config
    )
  }
}
```

and then you need to put your "new" graph and stuff in the `covid19_norsyss_vs_msis_lab_daily` (with lab) and convert `covid19_norsyss_vs_msis_weekly` back into the "normal graph" (without lab)

does that make sense?

File: ext/beta_kht/covid19.R (hunk 1 above)
Comment (raubreywhite) [reply ext/beta_kht/covid19.R:770]: max_right should be the maximum of the d_right and d_third

Comment (ybkamaleri) [reply ext/beta_kht/covid19.R]: second and third data are standardised with new variable `rescaled` and then make common `scaled_value` that applied to both

File: ext/beta_kht/covid19.R (hunk 3 above)
Comment (ybkamaleri) [reply ext/beta_kht/covid19.R]: Add `legend_labs` argument

File: ext/beta_kht/covid19.R (hunk 2 above)
Comment (ybkamaleri) [reply ext/beta_kht/covid19.R]: same

File: ext/beta_kht/covid19.R (hunk 4 above)
Comment (ybkamaleri) [reply ext/beta_kht/covid19.R]: Done

File: ext/beta_kht/covid19.R (hunk 5 above)
Comment (ybkamaleri) [reply ext/beta_kht/covid19.R]: I think I understood what you wanted it to be. But pls double check

File: ext/beta_kht/covid19.R [hunk 6]
```diff
@@ -763,12 +765,31 @@ covid19_plot_single <- function(
...
-  if(!is.null(d_third)){
-    d_third[, scaled_value := value / max_third * max_left]
-  }
+    minRight <- min(d_right$value)
+    maxRight <- max(d_right$value)
+    minThird <- min(d_third$value)
+    maxThird <- max(d_third$value)
+
+    ## Standardize value for d_right and d_third
+    tranRight <- coef(lm(c(minThird, maxThird) ~ c(minRight, maxRight)))
```
Comment (raubreywhite) [inline ext/beta_kht/covid19.R:776]: I think this has all gone a bit crazy :-)

```
  max_right <- max(d_right$value)
  max_right <- max(c(max_right, 5))
  if(!is.null(d_third)){
    max_right <- max(c(max_right, max(d_third$value))
    d_third[, scaled_value :=  value / max_right * max_left]
  }

  if(!is.null(d_right)){
    d_right[, scaled_value := value]
    d_right[, scaled_value := value / max_right * max_left]
  }
```

Comment (ybkamaleri) [reply ext/beta_kht/covid19.R]: Should have mentioned about it but I'm sure you will see it. Now I wasn't sure how did I think that the min and max of the two should be similar.

**Pull request merged**